src/makefile.all

```diff
@@ -644,16 +644,16 @@ $(WATT_DLL): $(OBJS) $(RESOURCE) $(LINKARG)
 $(STAT_LIB): $(OBJS) $(LIBARG)
 	$(AR) -q -b -c -pa $^@ @$(LIBARG)
 
--!include "$(OBJROOT)watt32.dep"
```

Comment:
What happened to this file now?

Reply:
Watcom wmake use dependency info included in .obj files by Watcom compilers therefore doesn't nee any dependency info in this file, It need not be used for watcom and wmake

Reply:
I see. Removed and pushed your stuff in this PR.
Hence closing. Thanks!